Source/WebCore/html/HTMLInputElement.idl

```diff
@@ -59,7 +59,7 @@ enum AutofillButtonType {
     [CEReactions=NotNeeded, Reflect] attribute DOMString formTarget;
     [CEReactions=NotNeeded] attribute unsigned long height;
     attribute boolean indeterminate;
-    [Conditional=DATALIST_ELEMENT] readonly attribute HTMLElement list;
+    readonly attribute HTMLElement list;
```

Comment:
Strange that this never had an `EnabledBySetting`, but not new.

Reply:
Yeah, I had added that initially but that regressed WK1 tests and I don't really want to risk that as part of this patch. Also not entirely sure it's worth it as this attribute (and corresponding element) should really be supported by all ports.